_data/themes/beach.yml

```diff
@@ -0,0 +1,91 @@
+
+# Color configurations
+colors:
+# changes content link color, hover color in nav, active highlight in nav, primary button bg color, return to top link color
+# usa-banner link color ???
+  primary: 'LightBlue'
+# content link hover color, usa-banner link hover color, primary button hover bg color, return to top link hover color
+  primary-darker: 'DarkBlue'
+# hero bg color, content section bg color
+  primary-darkest: 'Cornsilk'
+  primary-darkest-text-color: 'Black'
+  # primary-darker: 'darken($color-primary, 50%)'
+  # primary-darkest: 'darken($color-primary, 100%)'
+# hero alt heading color, hero link color, content section heading color
+  primary-alt: 'lightblue'
+  primary-alt-dark: 'purple'
+  primary-alt-darkest: 'blue'
+  # primary-alt-dark: 'darken($color-primary-alt, 5%)'
+  # primary-alt-darkest: 'darken($color-primary-alt, 10%)'
+  primary-alt-light: 'lighten($color-primary-alt, 5%)'
+  primary-alt-lightest: 'lighten($color-primary-alt, 10%)'
+  # secondary-dark: 'darken($color-secondary, 5%)'
+  # secondary-darkest: 'darken($color-secondary, 10%)'
+  secondary-light: 'lighten($color-secondary, 5%)'
+  secondary-lightest: 'lighten($color-secondary, 10%)'
+
+# header site title, active top nav link color, left nav link color, link color in primary footer and main footer
+# text in usa-banner ???
+  base: 'DarkSlateGray'
+
+# Navigation colors
+  navigation:
+# Top nav colors
+    top-nav-bg: 'DarkSlateGray'
+    top-nav-link: 'white'
+    top-nav-link-hover: 'green'
```

Comment:
Adding sub sections under colors like `navigation`, `usa-banner`, and `footer` to section them out a bit more.